prepare_llm_models.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -d "$1" ]; then
+  echo "Models directory $1 exists. Skipping downloading models."
+  exit 0
+fi
+
+echo "Downloading LLM testing models to directory $1"
+python3 -m venv .venv
```

Comment:
It might fail if the venv is already created. Maybe we should add existing venv target dependency instead in the Makefile?

Reply:
It doesn't fail, I tested running it few times in a row with different paths and it worked. I don't know about using same venv as we use for everything else. This one is pretty heavy and keeping it separate gives as easier option to split LLM test execution if we want to do that in the future.